### ISSUE TITLE
[KWS] Splitting Speech Command Dataset 

### ISSUE THREAD

**Issue by serser** [google-research/google-research#378]
@rybakov

Hi Oleg, I am a little confused about how you get datasets split. For instance In V2, we have 105843 samples in total, but only 36923/4445/4890 are used for train/val/test sets. 

The script [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/speech_commands/input_data.py) doesn't seem to handle the number of samples.

Best regards.

**Comment by serser** [comment]
Ah, I see it is decided here to have 10% unknown samples out of total set:
 https://github.com/tensorflow/tensorflow/blob/3c75664e72c40fc202fd986903cea39bd526f63d/tensorflow/examples/speech_commands/input_data.py#L318

**Comment by rybakov** [comment]
Yes, you already found the answer.

In more details:
[input_data.py](https://github.com/google-research/google-research/blob/master/kws_streaming/data/input_data.py) in kws_streaming was branched out of [input_data.py](https://github.com/tensorflow/tensorflow/blob/3c75664e72c40fc202fd986903cea39bd526f63d/tensorflow/examples/speech_commands/input_data.py#L318) so that test train data splits are the same and accuracy can be compared with previously published work.

[input_data.py](https://github.com/tensorflow/tensorflow/blob/3c75664e72c40fc202fd986903cea39bd526f63d/tensorflow/examples/speech_commands/input_data.py#L318)  is used to generate testing data list testing_list.txt which is part of archive [speech_commands_v0.02](https://storage.googleapis.com/download.tensorflow.org/data/speech_commands_v0.02.tar.gz)
By default 10% of the data are used for testing (listed in testing_list.txt).

Please feel free to look at [colab](https://github.com/google-research/google-research/blob/master/kws_streaming/colab/00_check_data.ipynb). There you can check data properties and validate that list of the data generated for testing by [input_data.py](https://github.com/google-research/google-research/blob/master/kws_streaming/data/input_data.py) is the same with the list of files in testing_list.txt.
Please let me know if you have more questions.
